price-calc/src/main/java/by/prohorov/price/entity/Price.java

```diff
@@ -0,0 +1,41 @@
+package by.prohorov.price.entity;
+
+import java.util.ArrayList;
+
+/**
+ * Created by Artsiom Prokharau 09.02.2021
+ */
+
+public class Price {
+
+    private Integer light;
```

Comment:
what happens when `Integer` will be nulls

Reply:
Created validatePrice